query_AIS_per_min.py

```diff
@@ -0,0 +1,91 @@
+def generate_data(dateStr='2020-12-09 00:01'):
+    import datetime
+    import json
+    import time
+    import urllib.request
+    import pytz  # time zones -- install via python -m pip install pytz
+
+    localDir = '/Users/homz/Downloads/'  # this is OUTSIDE the repo
+    print('local directory is ', localDir)
+    # dateStr = '12/09/2020 00:01'
+    timeWindow = 300
```

Comment:
in the latest data generation, changed to 30s to mitigate overlapping/duplications.